### DISCUSSION TITLE
Unraked Mod

### DISCUSSION THREAD

**Discussion by SatoCodingStuff** [ppy/osu#23052] [Ideas]
A mod which doesn't send the play to the leaderboard but dose bring the user to the score screen.
Normal osu! gameplay with no difference but doesn't submit the score

**Comment by Theighlin** [comment]
I think this is kinda a duplicate of #17121? I'd hope for it to be a mod and not a room but discussion is still on-going.

**Comment by SatoCodingStuff** [comment]
I see, looks like I didn't look up for the right words when I've search for it.
